src/main/java/net/minestom/server/utils/NamespaceID.java

```diff
@@ -62,6 +63,10 @@ public static NamespaceID from(String id) {
         return from(getDomain(id), getPath(id));
     }
 
+    public static NamespaceID from(Key key) {
```

Comment:
NamespaceID could use a `asKey()` function.

Reply:
Great idea - they can implement `Keyed` actually which would allow them to be used directly throughout the API.